src/styles/blocks/form.scss

```
@@ -0,0 +1,69 @@
+.form {
+  @include displayFlex(column, start);
+  gap: 16px;
+
+  &__input {
+    padding: 14px 16px;
+    box-sizing: border-box;
+
+    border: 1px solid $light-theme;
+    border-radius: 40px;
+    background: $light-theme;
+
+    font-family: "Poppins", sans-serif;
```

let's set font-family like this
```suggestion
* {
  font-family: "Poppins", sans-serif;
}
```